### PR TITLE
Passing IsMainFrame as parameter to the OnBeforeBrowse

### PR DESCRIPTION
**Pull request by myrnazhou**
Closing #829. Adding ```IsMainFrame``` to the ```OnBeforeBrowse``` method.

### CODE REVIEW

File: CefSharp/IRequestHandler.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public interface IRequestHandler
         /// <param name="request">the request object - cannot be modified in this callback</param>
         /// <param name="isRedirect">has the request been redirected</param>
```
Comment (amaitland) [inline CefSharp/IRequestHandler.cs]: Can you add additional <param/> please?

Comment (myrnazhou) [inline CefSharp/IRequestHandler.cs]: Added.

**Comment by amaitland** [comment]
Thanks :+1:

**Comment by lamerbug** [comment]
Since this commit my Cefsharp WinXP/VS2010 builds stopped working on Windows 7-8 (yes I got all building and working fine in XP), when I attach IRequestHandler to ChromiumWebBrowser it tells me that "Method not found Boolean CefSharp.IRequestHandler.OnBeforeBrowse (CefSharp.IWebBrowser, CefSharp.IRequest, Boolean, Boolean). But this method IS PRESENT both in CefSharp.IRequestHandler.cs and in my project's class implementing this interface. I'm wondering what can be wrong here?

**Pull request merged**